src/pages/ReportWelcomeMessagePage.js

```diff
@@ -0,0 +1,94 @@
+import React, {useCallback, useRef, useState} from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import {View} from 'react-native';
+import ExpensiMark from 'expensify-common/lib/ExpensiMark';
+import compose from '../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import ScreenWrapper from '../components/ScreenWrapper';
+import Navigation from '../libs/Navigation/Navigation';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import styles from '../styles/styles';
+import reportPropTypes from './reportPropTypes';
+import withReportOrNotFound from './home/report/withReportOrNotFound';
+import Text from '../components/Text';
+import TextInput from '../components/TextInput';
+import * as Report from '../libs/actions/Report';
+import ONYXKEYS from '../ONYXKEYS';
+import CONST from '../CONST';
+import FullPageNotFoundView from '../components/BlockingViews/FullPageNotFoundView';
+import Form from '../components/Form';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** The report currently being looked at */
+    report: reportPropTypes.isRequired,
+
+    /** Route params */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** Report ID passed via route r/:reportID/welcomeMessage */
+            reportID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+function ReportWelcomeMessagePage(props) {
+    const parser = new ExpensiMark();
+    const [welcomeMessage, setWelcomeMessage] = useState(parser.htmlToMarkdown(props.report.welcomeMessage));
+    const welcomeMessageInputRef = useRef(null);
+
+    const handleWelcomeMessageChange = useCallback((value) => {
+        setWelcomeMessage(value);
+    }, []);
+
+    const submitForm = useCallback(() => {
+        Report.updateWelcomeMessage(props.report.reportID, props.report.welcomeMessage, welcomeMessage);
+    }, [props.report.reportID, props.report.welcomeMessage, welcomeMessage]);
+
+    return (
+        <ScreenWrapper
```

Comment:
We should have passed:
- `shouldEnableMaxHeight` to avoid keyboard covering submit button (since we only have one input)
- `includeSafeAreaPaddingBottom={false}` because we have Form that already includes the safe area bottom padding